docs/grafana/README.md

```diff
@@ -15,12 +15,9 @@ To quickly test metrics locally (without installing Prometheus Grafana in the cl
     ```
 2. Get the serviceaccount's token to enable local access to metrics
     ```bash
-    TOKEN=$(kubectl get secrets -o=json -n ${NAMESPACE} | jq -r '[.items[] |
-      select (
-        .type == "kubernetes.io/service-account-token" and
-        .metadata.annotations."kubernetes.io/service-account.name" == "devworkspace-controller-serviceaccount")][0].data.token' \
-      | base64 --decode)
+    TOKEN=$(kubectl create token devworkspace-controller-serviceaccount -n ${NAMESPACE} --duration=1h | base64 --decode)
```

Comment:
`kubectl create token` returns a plain-text token; I think decoding is not needed.
```suggestion
    TOKEN=$(kubectl create token devworkspace-controller-serviceaccount -n ${NAMESPACE} --duration=1h)
```

Reply:
Good catch, updated